webhook.go

```diff
@@ -35,6 +35,7 @@ type Webhook struct {
 	UpdatedAt           *time.Time `json:"updated_at,omitempty"`
 	Fields              []string   `json:"fields"`
 	MetafieldNamespaces []string   `json:"metafield_namespaces"`
+	APIVersion          string     `json:"api_version"`
```

Comment:
Can you change this to `ApiVersion` please?

Reply:
Done